### DISCUSSION TITLE
Add utility to target Firefox only

### DISCUSSION THREAD

**Discussion by NatoBoram** [tailwindlabs/tailwindcss#10253] [Ideas]
There are differences between Firefox and Chrome that means we have to put browser-specific CSS code. I couldn't find a Tailwind utility to do that.

Example:

* The scrollbar in Chrome is included in the space calculation
* The scrollbar in Firefox is an overlay and is excluded in space calculation

![chrome-scrollbar](https://user-images.githubusercontent.com/10495562/210824782-fc4ba5e1-c9b9-4ce0-b78c-87e4872216f0.png) ![firefox-scrollbar](https://user-images.githubusercontent.com/10495562/210824780-5d1518f1-4a52-4258-a4bc-c929c6ba7789.png)

In both cases, I am using the same code:

```svelte
<div slot="content-summary" class="px-0 pb-3 md:pb-0" />
```

If I could, I would use Firefox-specific CSS to offset this differently across browsers.

```svelte
<div slot="content-summary" class="px-0 pb-1 firefox:pb-3 firefox:md:pb-0" />
```

Workaround:

```svelte
<div slot="content-summary" class="px-0 pb-1 content-summary-firefox" />

<style lang="postcss">
	:-moz-any(.content-summary-firefox) {
		@apply pb-3 md:pb-0;
	}
</style>
```

**Comment by adamwathan** [comment]
Hey! Yeah have tossed this around a bit and think it could be cool, just a bunch of decisions to make on the best way to do it or if we _really_ need it now that we have `@supports` support, but your example is a good example of where `@supports` isn't quite the right choice.

You can do this in user-land in the mean time if you want with a simple plugin, here's a demo:

https://play.tailwindcss.com/9zUopjQm94?file=config
